src/index.html

```diff
@@ -1,12 +1,360 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+    rel="shortcut icon"
+    href="./images/product-catalog.svg"
+    type="image"
+    >
+    <link rel="stylesheet" href="./styles/main.scss" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__image"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active"> apple </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link"> samsung </a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link"> smartphones </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active"> apple </a>
          </li>
          
          <li class="nav__item">
            <a href="#samsung" class="nav__link"> samsung </a>
          </li>
          
          <li class="nav__item">
            <a href="#smartphones" class="nav__link"> smartphones </a>
          </li>
```

read [check list](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) and fix html code style
![image](https://github.com/mate-academy/layout_catalog/assets/108538921/073226d4-c7e9-4b58-9fa8-5f62410d835f)